src/main/jhipster/src/main/webapp/i18n/en/feedback.json

```diff
@@ -3,7 +3,8 @@
     "title": "Feedback",
     "dont-show-again-checkbox": "Don't show again",
     "rating-text": {
-      "GERMPLASM_LIST": "How was your experience with the new list manager?"
+      "GERMPLASM_LIST": "How was your experience with the new list manager?",
```

Comment:
@cuenyad if the expeectation is to have one form in survey hero per feature then, it does not make much sense to send the rating-text from the BMS. This text can be defined in survey hero